website/static/js/registrationUtils.js

```diff
@@ -674,12 +1087,33 @@ RegistrationEditor.prototype.save = function() {
             schema_data: data
         });
     }
-    self.lastSaveRequest = request;
     request.fail(function() {
         $osf.growl('Problem saving draft', 'There was a problem saving this draft. Please try again, and if the problem persists please contact ' + SUPPORT_LINK + '.');
     });
     return request;
 };
+/**
+ * Makes ajax request for a project's contributors
+ */
+RegistrationEditor.prototype.makeContributorsRequest = function() {
+    var self = this;
+    var contributorsUrl = window.contextVars.node.urls.api + 'get_contributors/';
+    return $.getJSON(contributorsUrl);
+};
+/**
+ * Returns the `user_fullname` of each contributor attached to a node.
+ **/
+RegistrationEditor.prototype.getContributors = function() {
+    var self = this;
+    return self.makeContributorsRequest()
+        .then(function(data) {
+            return $.map(data.contributors, function(c) { return c.fullname; });
+        }).fail(function() {
+            $osf.growl('Could not retrieve contributors.', 'Please refresh the page or ' +
```

Comment:
Consider moving this text to `osfLanguage.js`, or reusing the predefined text snippets already available in that module.